patches/000-initial-setup.patch

```diff
@@ -954,14 +955,18 @@ index 3278a7ff30..e4484540c1 100644
  			testEverything(t, priv)
  		})
  	}
-@@ -629,6 +667,10 @@ func TestEncryptOAEP(t *testing.T) {
+@@ -629,6 +667,14 @@ func TestEncryptOAEP(t *testing.T) {
  	n := new(big.Int)
  	for i, test := range testEncryptOAEPData {
  		n.SetString(test.modulus, 16)
 +		if boring.Enabled && !boringtest.Supports(t, "RSA1024") && n.BitLen() < 2048 {
 +			t.Logf("skipping encryption tests with BoringCrypto: too short key: %d", n.BitLen())
 +			continue
 +		}
++ 	if boring.Enabled && i == 1 {
++ 		t.Log("skipping due to unsupported padding usage")
```

Comment:
This message is a bit confusing as OEAP is actually a supported padding scheme. Maybe something like "skipping as this test internally uses RSA encryption without padding"? Also good to have `!boringtest.Supports(t, "RSANoPadding")` as well?